fetchai/ledger/api/bootstrap.py

```diff
@@ -0,0 +1,51 @@
+import requests
+
+class NetworKUnavailableError(Exception)
+
+def server_from_name(network):
+    # Get list of active servers
+    params = {'active': 1}
+    servers_response = requests.get('https://bootstrap.fetch.ai/networks/', params=params)
+    if servers_response.status_code != 200:
+        raise requests.ConnectionError('Failed to get network status from bootstrap')
+
+    # Check requested server is on list
+    available_servers = [s['name'] for s in servers_response.json()]
+    if network not in available_servers:
+        raise NetworKUnavailableError('Requested network not present on network: {}'.format(network))
+
+    # Check network version
+    server_version = next(s for s in servers_response.json() if s['name'] == network)['versions']
+    # TODO: Not sure how to compare this string to our requirements
```

Comment:
We could check whether a network is potentially compatible here, but both the network and the API specify a range of versions. If the network is incompatible, this will eventually be caught when we finish instantiating the API.